### ISSUE TITLE
Elastic failed to parse out of range field [ps.handles.object] of type [long]

### ISSUE THREAD

**Issue by 0xThiebaut** [rabbitstack/fibratus#51]
When using Fibratus (`Version: 1.0.0 Commit:  Go compiler: go1.15.2`) with Elastic as output, the field mapping of  `ps.handles.object` seems to default to `long`, resulting in thousands of out of range errors. It might be worth considering defining the field to be an `unsigned_long` instead.

```json
{
  "level": "error",
  "msg": "failed to insert document 50: \\u0026{mapper_parsing_exception failed to parse field [ps.handles.object] of type [long] in document with id 'xlEk5XcB1qhpxSipG9lF'. Preview of field's value: '18446697357984943568'     false map[reason:Numeric value (18446697357984943568) out of range of long (-9223372036854775808 - 9223372036854775807)\n at [Source: (byte[])\"{\"seq\": 933,\"pid\": 2908,\"tid\": 8688,\"cpu\": 0,\"name\": \"CloseHandle\",\"category\": \"handle\",\"description\": \"Closes the handle\",\"host\": \"REDACTED\",\"timestamp\": \"2021-02-27T21:17:31.4245833+01:00\",\"kparams\": {\"handle_object\": \"ffffc401d2a183f0\",\"handle_id\": 2280,\"handle_name\": \"HKEY_LOCAL_MACHINE\\\\SOFTWARE\\\\Microsoft\\\\AMSI\\\\Providers\",\"handle_type\": \"Key\"},\"meta\": {},\"ps\": {\"pid\": 2908,\"ppid\": 2908,\"name\": \"svchost.exe\",\"comm\": \"C:\\\\Windows\\\\system32\\\\svchost.exe -k netsvcs -p -s Winmgmt\",\"exe\": \"[truncated 33462 bytes]; line: 1, column: 32685] type:i_o_exception] [] [] []   \\u003cnil\\u003e}",
  "source": "elasticsearch/elasticsearch.go:143",
  "time": "2021-02-27T21:18:05+01:00"
}
```

While reporting this issue I also came across hundreds of the following entry which might indicate another unrelated issue.

```json
{
  "level": "error",
  "msg": "interceptor chain error(s) occurred: \"handle\" interceptor failed with error: expected CloseHandle kernel event but got CreateHandle",
  "source": "kstream/kstreamc.go:392",
  "time": "2021-02-27T21:42:28+01:00"
}
```

Thanks for the fun tool!

**Comment by rabbitstack** [comment]
Hey @0xThiebaut 

Thanks for reporting! I'll look into this. It should be fixed in the 1.2.0 version which I'm planning to release in a few weeks.

Nevertheless, I'm not sure why did Elasticsearch opted for assigning the long type to this field. It is not defined in the internal mapping, but looks like I'll have to force it anyway.

**Comment by rabbitstack** [comment]
@0xThiebaut this bug was fixed. I'll publish a new release soon, but if you can't wait until then, you can grab the "nightly"  binary from Actions artifacts.

Regards,
